### ISSUE TITLE
What’s the reason behind the name change?

### ISSUE THREAD

**Issue by loudgazelle** [rudders/homebridge-wemo#29]
What’s the reason behind the name change? Is there functionality that the old plugin couldn’t provide due to its name, or a limitation in that plugin that required moving to a plugin with a new name?

According to the notice on the [homebridge-platform-wemo](https://github.com/bwp91/homebridge-platform-wemo) page the process of moving to the new plugin requires a decent amount of effort from the end users:
> If you are already using homebridge-platform-wemo you should consider moving to homebridge-wemo. You will need to reconfigure your HomeKit accessories from scratch - including their names, rooms, scenes and automations.

… but there’s no explanation about the reasons that prompted the name change which requires this effort from the end users. 

A name change is entirely within your rights as an uncompensated developer and maintainer of this plugin, but that change does lead to a not insignificant amount of work for all the users of this plugin if they want to stay current, so it’d be useful to have some insight into the reasons that led to the change.

**Comment by loudgazelle** [comment]
#26 is related

**Comment by bwp91** [comment]
Hi,

`homebridge-platform-wemo` was an unmaintained plugin being held by homebridge looking for a maintainer - I took this on about 10 months ago.

The `homebridge-wemo` repo was also unmaintained, and this was offered to me a few weeks ago, so I decided to take it.

Of the two repo names, I think `homebridge-wemo` is cleaner and in-line with other plugins including others that I maintain.

In the future my focus will be on `homebridge-wemo` so naturally I have recommended users to switch to this plugin, but there is no urgency to switch as `homebridge-platform-wemo` is not suddenly going to stop working. In fact you don't *have* to switch at all.
